python/app/ui/resources_rc.py

```diff
@@ -12,7 +12,10 @@
     try:
         from PySide2 import QtCore
     except ImportError:
-        from PySide import QtCore
+        try:
+            from PySide6 import QtCore
+        except ImportError:
+            from PySide import QtCore
```

Comment:
Similar comment as in shotgunsoftware/tk-maya#106, I wonder why we don't use the tk-core's QtImporter here? Instead of duplicating the strategy and making rooms for bugs or difference of behaviors.

Reply:
If you see line 10 on this same file, it's using the tk-core importer. However, I don't know the reason why there are direct imports from PySide. Maybe something related to CI?

Reply:
Ok I have a concrete answer to your question. These PySide* imports are needed for this [test suite](https://github.com/shotgunsoftware/tk-multi-pythonconsole/blob/master/tests/test_standalone_app.py#L26), which tests this component without `sgtk` only.